src/foam/nanos/crunch/ReputDependentUCJs.js

```diff
@@ -80,7 +80,7 @@ foam.CLASS({
             for ( UserCapabilityJunction ucjToReput : ucjsToReput ) {
               if ( ucjToReput.getStatus() == CapabilityJunctionStatus.GRANTED ) {
                 if ( ucj.getIsInGracePeriod() ) ucjToReput.setIsInGracePeriod(true);
-                if ( ucj.getIsRenewable() ) ucjToReput.setIsRenewable(true);
+                if ( ucj.getIsInRenewablePeriod() ) ucjToReput.setIsInRenewablePeriod(true);
               }
```

Comment:
Note for this change: `isRenewable` doesn't need to be explicitly set since this is computed based on the value of `IsInGracePeriod`, `isExpired` and `isInRenewablePeriod` in a factory. On the other hand `isInRenewablePeriod` needs to be set here.